tests/unit/bucket/bucket_mem_api.js

```diff
@@ -294,14 +294,9 @@ describe('stress test for bucket API', function describe() {
         }
 
         // Populate keys array with all keys including prefixes
-        let key;
-        for (key in data) {
-            if (data.hasOwnProperty(key)) {
-                for (let k = 0; k < data[key].length; k++) {
-                    keys.push(key + delimiter + data[key][k]);
-                }
-            }
-        }
+        Object.keys(data).forEach(dkey => {
+            data[dkey].map(key => keys.push(dkey + delimiter + key));
```

Comment:
I think you still want to use a forEach here; since you aren't using the object returned by `map`